keepercommander/commands/discoveryrotation.py

```diff
@@ -2134,7 +2134,7 @@ def execute(self, params, **kwargs):
             encrypted_transmission_key=encrypted_transmission_key,
             encrypted_session_token=encrypted_session_token)
 
-        print_router_response(router_response, 'job_info', conversation_id)
+        print_router_response(router_response, 'job_info', conversation_id, gateway_uid=gateway_uid)
```

Comment:
I can see this function is called from 9 commander's commands
only one command adds gateway parameter `pam action rotate`

Some other commands also have gateway context like `pam action gateway-info`

Are we sure this fix is not needed in other commands that shows scheduled commands?